configure

```diff
@@ -1294,6 +1300,12 @@ arm-riscos)
 	_host_os=riscos
 	_host_cpu=arm
 	;;
+raspberrypi)
+	_host_os=linux
+	_host_cpu=arm
+	_host_alias=arm-linux-gnueabihf
+	_backend=raspberrypi
```

Comment:
That's the wrong place to set _backend (if you look around, you'll see that no other case does it here. Instead, it should be set in the big case selection starting somewhere around line 2485.